dlrover/python/common/constants.py

```diff
@@ -96,6 +96,37 @@ class NodeEventType(object):
     NODE_CHECK_FAILED = "NODE_CHECK_FAILED"
 
 
+class GpuMetricType(object):
```

Comment:
Maybe just `GPUMetrics`? the same for `NPUMetrics`. In addition, since those constants are under class NPUMetrics and GPUMetrice, it is unnecessary to start with GPU_XXX, just XXX will be ok. The reference will be GPUMEtrics.Free_MEM.

Reply:
GPU and NPU have many different metric type